apps/launchpad/app/components/points-card/points-card.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+
+import {
+  Text,
+  TextVariant,
+  Tooltip,
+  TooltipContent,
+  TooltipProvider,
+  TooltipTrigger,
+} from '@0xintuition/1ui'
+
+import { truncateNumber } from '@lib/utils/misc'
+
+interface PointsEarnedCardProps {
+  totalPoints: number
+  activities: Array<{
+    name: string
+    points: number
+    disabled?: boolean
+  }>
+}
+
+const PointsRow: React.FC<{
+  name: string
+  points: number
+  totalPoints: number
+  disabled?: boolean
+}> = ({ name, points, totalPoints, disabled = false }) => {
+  return (
+    <TooltipProvider>
+      <Tooltip>
+        <TooltipTrigger asChild>
+          <div className="grid grid-cols-7 items-center gap-2">
+            <Text
+              variant={TextVariant.body}
+              className={`col-span-1 ${disabled ? 'text-primary/40' : ''}`}
+            >
+              {name}
+            </Text>
+            <div className="col-span-5 h-1.5 w-full bg-muted rounded-sm mx-auto">
+              <div
+                className="h-full bg-primary rounded-sm"
+                style={{
+                  width: `${totalPoints === 0 ? 0 : (points / totalPoints) * 100}%`,
+                }}
+              />
+            </div>
+
+            <Text
+              variant={TextVariant.body}
+              className={`col-span-1 text-right ${disabled ? 'text-primary/40' : ''}`}
+            >
+              {truncateNumber(points)}
+            </Text>
+          </div>
+        </TooltipTrigger>
+        <TooltipContent>
+          {disabled
+            ? 'Coming Soon'
+            : `${(totalPoints === 0 ? 0 : (points / totalPoints) * 100).toFixed(1)}%`}
+        </TooltipContent>
+      </Tooltip>
+    </TooltipProvider>
+  )
+}
+
+export const PointsEarnedCard: React.FC<PointsEarnedCardProps> = ({
+  totalPoints,
+  activities,
+}) => {
+  return (
+    <div className="flex flex-col theme-border rounded-lg p-6 gap-4 bg-black">
+      {/* <div className="flex flex-col gap-1">
```

Comment:
If we don't need we can remove this commented code

Reply:
Sure, I don't think we will since the total points is listed in the component above it. We can add it back in later if needed.